.github/workflows/test.yaml

```diff
@@ -6,17 +6,6 @@ on:
   - pull_request
 
 jobs:
-  codestyle:
-    name: Codestyle
-    runs-on: ubuntu-18.04
-    container: thalia/concrexit-dependencies
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Run black
-        run: make blackcheck
-
   linting:
     name: Linting
```

Comment:
If this contains style checks then the name should reflect that.

In any case this removes the parrallellism of the codestyle and the linting jobs. And it will show as 1 check in the overview in PRs, I really don't like that personally. I like to see exactly what failed by just glancing over the checks instead of having to open them.

Reply:
For parallelism I could put all three in a seperate check?

Reply:
That is what I have done now